package/MDAnalysis/analysis/rms.py

```diff
@@ -335,43 +328,53 @@ def __init__(self, traj, reference=None, select='all',
         .. versionadded:: 0.7.7
         .. versionchanged:: 0.8
            *groupselections* added
+        .. versionchanged: 0.15.1
+            Refactor to fit with Bauhaus API
```

Comment:
You should use another name in the docs. Bauhaus is something we use internal for development. it won't say much to anybody using the class.  Just say what changed in the API